andela/ah-92explorers-frontend
# #166789950 Run Codeclimate Test Coverage

#### What does this PR do?
- Configure codeclimate test coverage
#### Description of Task to be completed
- Enables run code climate test coverage 
#### How should this be manually tested?
```bash
git clone https://github.com/andela/ah-92explorers-frontend.git
cd ah-92explorers-frontend
git checkout ch-run-codeclimate-test-cov-166789950
```
- Open README file to see the badges
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166789950](https://www.pivotaltracker.com/story/show/166789950)
#### Screenshots (if appropriate)
<img width="828" alt="Screen Shot 2019-08-16 at 16 08 01" src="https://user-images.githubusercontent.com/30776949/63173410-256f4580-c040-11e9-85c8-4588b13a7e97.png">

#### Checklist
